db/migrate/20221017130819_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name, null: false, default: "user"
+
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] there is no migration file for recipes so It will not going to be any recipe table created in the DB so the app will not render any recipe can you please add the migration file for recipe